packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -222,7 +223,7 @@ MeetingInfoUtil.getResourceUrl = (type, value) => {
       method = HTTP_VERBS.PUT;
       break;
     case _LOCUS_ID_:
-      resource = `/${LOCI}/${value}/${MEETINGINFO}`;
+      uri = `${value}/${MEETINGINFO}`;
```

Comment:
Does this mean that a locus id request has a full locus url in it?

Reply:
yes